src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -921,6 +940,45 @@ private MenuBar createMenu() {
         return menu;
     }
 
+    private Group createTaskIndicator() {
+        ProgressIndicator indicator = new ProgressIndicator();
+        indicator.getStyleClass().setAll("progress-indicator");
+        indicator.progressProperty().bind(stateManager.tasksProgressBinding);
```

Comment:
Also add `Background Tasks` as tooltip (if supported)?

Reply:
I actually wanted to do that already, but forgot.

Reply:
https://github.com/JabRef/jabref/pull/6443/commits/cf10859542c790e170866698f57a6fc8a704f7df